src/ui/public/doc_table/doc_table.js

```diff
@@ -136,9 +136,10 @@ uiModules.get('kibana')
             let inspectorRequest = undefined;
             if (_.has($scope, 'inspectorAdapters.requests')) {
               $scope.inspectorAdapters.requests.reset();
-              inspectorRequest = $scope.inspectorAdapters.requests.start('Data', {
-                description: `This request queries Elasticsearch to fetch the data for the search.`,
+              const description = i18n('common.ui.docTable.inspectorRequestDescription', {
+                defaultMessage: 'This request queries Elasticsearch to fetch the data for the search!',
               });
+              inspectorRequest = $scope.inspectorAdapters.requests.start('Data', { description });
```

Comment:
Actually the title `Data` is also a user visible string here, so it should be translated as well.